crengine/include/crsetup.h

```diff
@@ -131,7 +131,9 @@
 #else
 #define USE_FREETYPE                         1
 #define USE_HARFBUZZ                         1
+#ifndef _MSC_VER
```

Comment:
This kind of preprocessor stuff strikes me as more a matter for CMakeLists? Put another way, surely MSC isn't incompatible with FriBidi, just a little harder to setup?

Reply:
Yes, I agree but I don't wan to mess with this CMake stuff. We use it as a static library and it doesn't work well with MSVC. If someone will be interested to build CR with fribidi he will have to fix it properly.